Jenkinsfile

```diff
@@ -124,161 +124,160 @@ def sendNotifications(String buildStatus = '', String stageName = '', String git
-			gitCommitId = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+					// Get the current commit id
+					gitCommitId = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
 
-			// Get the remote repo url. This assumes we are using git+https not git+ssh.
-			gitRepoUrl = sh(returnStdout: true, script: 'git config --get remote.origin.url').trim().replaceAll(/\.git$/,"")
+					// Get the remote repo url. This assumes we are using git+https not git+ssh.
+					gitRepoUrl = sh(returnStdout: true, script: 'git config --get remote.origin.url').trim().replaceAll(/\.git$/,"")
 
-			// Send notifications that the build has started
-			sendNotifications('STARTED', currentStage, gitCommitId, gitRepoUrl)
-		}
-	}
-
-	/* This stage switches the gitBranchName (needed for our CCS downsream stages) 
-	value if the build is a PR as the BRANCH_NAME var is populated with the build 
-	name during PR builds. 
-	*/
-	stage('Set Branch Name') {
-		currentStage = "${env.STAGE_NAME}"
-		script {
-			if (env.BRANCH_NAME.startsWith('PR')) {
-				gitBranchName = env.CHANGE_BRANCH
-			} else {
-				gitBranchName = env.BRANCH_NAME
+					// Send notifications that the build has started
+					sendNotifications('STARTED', currentStage, gitCommitId, gitRepoUrl)
+				}
 			}
-		}
-	}
 
-	stage('Build Platinum AMI') {
-		currentStage = "${env.STAGE_NAME}"
-		if (params.build_platinum || amiIds.platinumAmiId == null) {
-			milestone(label: 'stage_build_platinum_ami_start')
-			node {
-				amiIds = scriptForDeploys.buildPlatinumAmi(amiIds)
+			/* This stage switches the gitBranchName (needed for our CCS downsream stages) 
+			value if the build is a PR as the BRANCH_NAME var is populated with the build 
+			name during PR builds. 
+			*/
+			stage('Set Branch Name') {
+				currentStage = "${env.STAGE_NAME}"
+				script {
+					if (env.BRANCH_NAME.startsWith('PR')) {
+						gitBranchName = env.CHANGE_BRANCH
+					} else {
+						gitBranchName = env.BRANCH_NAME
+					}
+				}
 			}
-		} else {
-			org.jenkinsci.plugins.pipeline.modeldefinition.Utils.markStageSkippedForConditional('Build Platinum AMI')
-		}
-	}
 
-	stage('Deploy mgmt') {
-		currentStage = "${env.STAGE_NAME}"
-		if (canDeployToProdEnvs && params.deploy_management) {
-			lock(resource: 'env_management', inversePrecendence: true) {
-				milestone(label: 'stage_management_jenkins_start')
+			stage('Build Platinum AMI') {
+				currentStage = "${env.STAGE_NAME}"
+				if (params.build_platinum || amiIds.platinumAmiId == null) {
+					milestone(label: 'stage_build_platinum_ami_start')
 
-				node {
-					scriptForDeploy.deployManagement(amiIds)
+					container('bfd-cbc-build') {
+						amiIds = scriptForDeploys.buildPlatinumAmi(amiIds)
+					}
+				} else {
+					org.jenkinsci.plugins.pipeline.modeldefinition.Utils.markStageSkippedForConditional('Build Platinum AMI')
 				}
 			}
-		} else {
-			org.jenkinsci.plugins.pipeline.modeldefinition.Utils.markStageSkippedForConditional('Deploy mgmt')
-		}
-	}
```

Comment:
We remove deploy management here as it was never actually functional, and not needed anymore as we don't manage Jenkins in CB Core!